Readme.md

```diff
@@ -17,6 +17,12 @@ Information on HackRF and purchasing HackRF: https://greatscottgadgets.com/hackr
 
 Documentation for HackRF can be viewed on [Read the Docs](https://hackrf.readthedocs.io/en/latest/). The raw documenation files for HackRF are in the [docs folder](https://github.com/mossmann/hackrf/tree/master/docs) in this repository and can be built locally by installing [Sphinx Docs](https://www.sphinx-doc.org/en/master/usage/installation.html) and running `make html`. Documentation changes can be submitted through pull request and suggestions can be made as GitHub issues. 
 
+To create a PDF of the HackRF One documentation from the HackRf repository:
+* navigate to hackrf/docs on command line
+* run the command `make latex`
+* navigate to hackrf/docs/latex
```

Comment:
Based on my testing, this directory change is incorrect and unnecessary.

Reply:
Change made